pkg/master/master.go

```diff
@@ -549,6 +549,46 @@ func (m *Master) InstallSwaggerAPI() {
 	swagger.RegisterSwaggerService(swaggerConfig, m.handlerContainer)
 }
 
+// Picks the best available address for an internal health check
+// TODO: Create utility function, combining this an GetHostIP in kubelet.go?
+func pickAddress(node *api.Node) string {
```

Comment:
I would generally assume that #7092 is aspirational until someone is signed up to work on it.